### PR TITLE
Fix/harden deps

### PR DESCRIPTION
**Pull request by samzilverberg**

### CODE REVIEW

File: lib/runner.js [hunk 1]
```diff
@@ -89,10 +89,11 @@ __private__.addProcess = function(number, queue) {
 
 
 __private__.shutdown = function() {
-  var endMoment = moment.utc();
+  var endMoment = moment.utc(),
+    totalDuration = moment(endMoment.diff(__private__.startDate)).format('HH:mm:ss');
```
Comment (kirrg001) [inline lib/runner.js]: 1. **private**.startDate is also a moment.
   can you rename to have consistency?
   **private**.startDateMoment
2. why do you need to wrap into moment again? moment(endMoment.diff(....))?

File: package.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
   "dependencies": {
-    "async": "^1.4.0",
-    "lodash": "^3.10.0",
+    "async": "1.4.0",
+    "lodash": "3.10.0",
     "mocha": "2.3.4",
-    "moment": "^2.10.3",
-    "optimist": "^0.6.1"
+    "moment": "2.11.1",
+    "optimist": "0.6.1"
```
Comment (kirrg001) [inline package.json:10]: +1 thanks for pinning

File: lib/runner.js (hunk 1 above)
Comment (samzilverberg) [inline lib/runner.js]: :+1: about rename

the diff is a number

Comment (kirrg001) [inline lib/runner.js]: +1 thx for explaining.

**Pull request merged**